modules/ROOT/pages/building-soap-proxy.adoc

```diff
@@ -81,6 +81,24 @@ You can resolve this issue by adding the appropriate imports in the WSDL file. F
 </types>
 ----
 
```

Comment:
So...does the intro text make clear that there are two METHODS of retrieving WSDL: either by using user credentials or by using TLS?

Reply:
Done! Thanks!